host_vars/pulsar-high-mem1/pulsar-high-mem1.yml

```diff
@@ -68,20 +68,6 @@ slurm_partitions:
       MaxTime: INFINITE
       State: UP
 
-slurm_config:
```

Comment:
need slurm_controller_host for this and other high memory pulsars, pulsar blast etc?